dev-docs/modules/relevadRtdProvider.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: page_v2
+title: Relevad RTD Provider
+display_name: Relevad RTD Provider
+description: Contextual Categories and Segments, cookieless and privacy-first.
+page_type: module
+module_type: rtd
+module_code : airgridRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# Relevad RTD Provider
+
+Relevad is a contextual semantic analytics company. Our privacy-first, cookieless contextual categorization, segmentation, and keyword generation platform is designed to help publishers and advertisers optimize targeting and increase ad inventory yield.
+
+This real-time data processing module provides quality contextual IAB categories and segments to prebid.js auction bidders. Every category and segment comes with a score measuring its relevance to the publisher’s web page. Publishers may configure their preferred minimum score setting with the module parameter “minscore”. Relevad service does not use browser cookies and is fully GDPR compliant.
+
+{:.no_toc}
+* TOC
+{:toc}
+
+## Usage
+
+Compile the Relevad RTD module (`relevaddRtdProvider`) into your Prebid build, along with the parent RTD Module (`rtdModule`):
+
+`gulp build --modules=rtdModule,airgridRtdProvider,appnexusBidAdapter`
```

Comment:
did you mean airgrid here? I assume you mean relevaddRtd?

Reply:
Yes, fixed
Thank you very much!